lib/kubeclient/common.rb

```diff
@@ -227,14 +227,11 @@ def create_entity(entity_type, entity_config, klass)
     end
 
     def update_entity(entity_type, entity_config)
-      name = entity_config.metadata.name
-      # to_hash should be called because of issue #9 in recursive open
-      # struct
-      hash = entity_config.to_hash
-      ns_prefix = build_namespace_prefix(entity_config.metadata['table'][:namespace])
+      name      = entity_config[:metadata][:name]
```

Comment:
`metadata.name` was working, but my thought was that all references into `entity_config` should be consistent.  In the original version of the PR the reference to ns_prefix was
```
build_namespace_prefix(entity_config.metadata.namespace)
```
but seemed like [the preference](https://github.com/abonas/kubeclient/pull/150#discussion_r49056553) was towards using `[]` rather than method references.